composer.json

```diff
@@ -1,22 +1,18 @@
 {
     "name": "outeredge/edge-magento-module-allconfigurableoptions",
```

Comment:
Rename to `outeredge/magento-outofstockoptions-module`

Reply:
The rename changes only need to be in the composer.json rite?